tests/git-notes.test.ts

```diff
@@ -38,9 +38,10 @@ test("set/get notes", async () => {
 
     const commit = await revParse(notes.notesRef, notes.workDir);
     expect(commit).not.toBeUndefined();
-    expect(await notes.appendCommitNote(commit!, "1")).toEqual("");
-    expect(await notes.getCommitNotes(commit!)).toEqual("1");
-    expect(await notes.appendCommitNote(commit!, "2")).toEqual("");
-    expect(await notes.getCommitNotes(commit!)).toEqual("1\n\n2");
-    expect(await notes.getLastCommitNote(commit!)).toEqual("2");
+    if (!commit) throw new Error(`assertion failed`); // avoid TS2532
```

Comment:
Does this make line 40 redundant?

Reply:
As far as ESLint is concerned: yes. Unfortunately, `tsc` would fail without this line (raising error [TS2532](https://github.com/microsoft/TypeScript/blob/4d85f03afa2fe5fbf015f017a832897ee8ed9a95/src/compiler/diagnosticMessages.json#L2132-L2135)).